README.md

```diff
@@ -1,7 +1,85 @@
 The goal of *mingwpy* project is to provide a free toolchain for building Python extensions for Windows OS, and make sure that those extensions are compatible with CPython compiled with native Visual Studio.
 
-*mingwpy* toolchain contains customized GCC compiler of the [GNU toolchain](https://en.wikipedia.org/wiki/GNU_toolchain) based on the [Mingw-w64](http://mingw-w64.org) and the [mingw-builds](https://github.com/niXman/mingw-builds) projects.
+*mingwpy* toolchain contains customized gcc, g++ and gfortran compilers of the [GNU toolchain](https://en.wikipedia.org/wiki/GNU_toolchain) based on the [Mingw-w64](http://mingw-w64.org) and the [mingw-builds](https://github.com/niXman/mingw-builds) projects.
 
  * Documentation: https://mingwpy.github.io/
  * Discussions: https://groups.google.com/forum/#!forum/mingwpy
- * Development: https://github.com/mingwpy/mingwpy/blob/master/mingwpy_builds_readme.md
+
+### Installing built mingwpy toolchain with pip
+
+Experimental builds of mingwpy for Python 2.7 and 3.4 are available for testing purposes:
+
+    pip install -i https://pypi.anaconda.org/carlkl/simple mingwpy
+
+Note: [Python 3.5 is not supported yet](https://mingwpy.github.io/issues.html#choice-of-msvc-runtime).
+
+### Using new toolchain with CPython
+
+To use the toolchain for CPython, see
+[readme_specs.md](https://github.com/mingwpy/mingwpy/blob/master/specs/readme_specs.md).
```

Comment:
This should be moved at the end of the section for building the toolchain from source.

The correct specs file is already integrated in the pre-built toolchain installed with pip. When installing a toolchain with pip it can only be used to build extensions for the version and architecture of the Python interpreter used by pip.